test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -229,11 +229,11 @@ eq_tends(
 
 eq_tends(pv::PV, m::EDMF, ::Source) where {PV} = ()
 
-eq_tends(pv::PV, m::EDMF, ::Source) where {PV <: EDMFPrognosticVariable} =
-    (EntrDetr{PV}(),)
+eq_tends(pv::PV, m::EDMF, ::Source) where {PV <: EDMFPrognosticVariable} = ()
+    # (EntrDetr{PV}(),)
```

Comment:
Note to self: only for testing

Reply:
this entire branch is just for testing now